index.bs

```diff
@@ -424,18 +455,52 @@ For each <a>element proxy</a> for that element, perform the following steps:
     7. For each property in |inputProperties|, if the property’s computed value has changed,
         set the <a>animation requested flag</a> on the |animator| to <a>frame-requested</a>.
 
-The timelines present in an animator's <a>animator timeline list</a> can also affect the
-<a>animation requested flag</a>:
+Animator specifies a DocumentTimeline {#requesting-animation-frames-document-timeline}
+-------------------------------------
+
+If a {{DocumentTimelineOptions}} instance is present in an animator's <a>animator timeline list</a>
+then that animator's <a>animation requested flag</a> <em>must</em> be set to <a>frame-requested</a>
+at the start of each animation frame.
+
+Animator specifies a ScrollTimeline {#requesting-animation-frames-scroll-timeline}
+-----------------------------------
+
+If a {{ScrollTimelineOptions}} instance is present in an animator's <a>animator timeline list</a>
+then the user agent <em>must</em> run the following steps when the scroll position of the nearest
+non-visible overflow ancestor to the animator’s <a>root element</a> changes:
```

Comment:
More of a note, we should really define this ancestor overflow element somewhere so that we don't accidentally change one copy of this description without updating the others.